Framework/PubNub/Info.plist

```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>PubNub</string>
 	<key>CFBundleGetInfoString</key>
-	<string>4.11.1</string>
+	<string>4.11.0</string>
```

Comment:
Downgrade?

Reply:
This is not the only place, but versions will be bumped during release.